src/test/java/seedu/uninurse/logic/parser/EditRemarkCommandParserTest.java

```diff
@@ -0,0 +1,72 @@
+package seedu.uninurse.logic.parser;
+
+import static seedu.uninurse.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.uninurse.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.uninurse.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.uninurse.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.uninurse.testutil.Assert.assertThrows;
+import static seedu.uninurse.testutil.TypicalIndexes.INDEX_FIRST_ATTRIBUTE;
+import static seedu.uninurse.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.uninurse.logic.commands.EditRemarkCommand;
+import seedu.uninurse.model.remark.Remark;
+
+/**
+ * Contains unit tests for {@code EditRemarkCommandParser}.
+ */
+class EditRemarkCommandParserTest {
+    private static final String REMARK_STUB = "Some remark";
```

Comment:
instead of declaring a `REMARK_STUB` here, perhaps you could use one of the `TYPICAL_REMARK_XX` strings you defined in `TYPICAL_REMARKS`.

Reply:
Yup I thought of that but since I was reusing some code from `EditTaskCommandParserTest` and `EditTaskCommandTest` I plan to leave this as it is for now and change it when we standardize how we write test cases in v1.4.